bin/lib/Logos/Generator/Base/Class.pm

```diff
@@ -49,10 +49,10 @@ sub initializers {
 	my $self = shift;
 	my $class = shift;
 	my $return = "";
-	for(@{$class->methods}) {
+	for(@{$class->properties}) {
```

Comment:
Why this swap?

Reply:
Look at the message in my last commit

Reply:
> Initializing %properties before method hooks allows properties to be hooked by methods.

Why would you do that?

Reply:
#41

Reply:
It'd be more appropriate to not generate the property methods if the class already has them, no?

Reply:
I now see that hooking the generated methods causes less work to be done: `%orig` would call the generated implementation of the g/setter with no special handling for the method and would not require adding `%new` before it. Let's do it this way.